tests/publish/ninjs_formatter_test.py

```diff
@@ -311,7 +311,7 @@ def test_item_with_usable_associations(self):
         formatted = json.loads(doc)
         self.assertIn('associations', formatted)
         self.assertIn('image', formatted['associations'])
-        image = formatted['associations']['image']
+        image = formatted['associations']['image'][0]
```

Comment:
I don't think they really use lists in the standard:

http://www.iptc.org/std/ninjs/ninjs-schema_1.0.json